pkg/compiler/debug.go

```diff
@@ -426,9 +426,6 @@ func parsePairJSON(data []byte, sep string) (string, string, error) {
 // ConvertToManifest converts contract to the manifest.Manifest struct for debugger.
 // Note: manifest is taken from the external source, however it can be generated ad-hoc. See #1038.
 func (di *DebugInfo) ConvertToManifest(o *Options) (*manifest.Manifest, error) {
-	if di.MainPkg == "" {
```

Comment:
We need this check to define whether the exported compiled method should be included in the manifest. There might not be the main method, but it's an error if there's no main package.

Reply:
This could be null when we searched for packages containing `Main` method.
Now main package is the one we provide to `contract compile` and it is always set in `emitDebugInfo`.

Reply:
BTW, after contract hash was moved, we don't need the `contract` parameter from `emitDebugInfo(contract []byte)`  anymore.